setup.cfg

```diff
@@ -24,10 +24,6 @@ replace = version='{new_version}'
 search = __version__ = '{current_version}'
 replace = __version__ = '{new_version}'
 
-[bumpversion:file:conda/meta.yaml]
```

Comment:
Why was this removed?

Reply:
we deleted the conda folder a while ago and didn't update this